decred/decred/wallet/wallet.py

```diff
@@ -68,10 +53,8 @@ def __init__(self, path):
         self.mgrCache = {}
         # The best block.
         self.users = 0
-        # User provided callbacks for certain events.
-        self.mtx = Mutex()
```

Comment:
DB access is now thread-safe. There may be reasons to consider locking for certain actions, but lock will be on the account, not the wallet.